recipes/leparagliding/build.sh

```diff
@@ -0,0 +1,3 @@
+${GFORTRAN} leparagliding.f
```

Comment:
I think this should be using `$FC`

Reply:
does it work for windows too?

Reply:
@scopatz  I think no (at least there is no such variable in the build). So I guess this should be ready.

Reply:
It should.  It is in the activation scripts for the compiler

Reply:
I tried this command for windows:
`%FC% leparagliding.f`

but I see this in the logs:
`%SRC_DIR%>leparagliding.f`

so I return to calling `gfortran` directly

Reply:
@isuruf - shouldn't `%FC%` work for windows too?

Reply:
No, `%FC%` is not set.

Reply:
so feel free to merge : )

Reply:
@conda-forge/core